RPC/Sources/RPC/Handlers/ChainHandlers.swift

```diff
@@ -67,6 +69,28 @@ public enum ChainHandlers {
         }
     }
 
+    public struct SubmitWorkPackage: RPCHandler {
+        public typealias Request = Request1<Data?>
```

Comment:
```suggestion
        public typealias Request = Request1<Data>
```

it is required parameter

Reply:
changed